tests/unit_tests/corsika/test_corsika_runner.py

```diff
@@ -43,7 +43,7 @@ def corsika_runner(corsika_config_data, io_handler, simtel_path, db_config):
 @pytest.fixture
 def corsika_file(io_handler):
     corsika_file = io_handler.get_input_data_file(
-        file_name="run1_proton_za20deg_azm0deg_North_1LST_test-lst-array.corsika.zst", test=True
+        file_name="run1_proton_za20deg_azm0deg_north_1lst_test-lst-array.corsika.zst", test=True
```

Comment:
Please do not change the name of the file, it is produced with the specific naming and capitalisation and should not change unless there is good reason for it. In this case it seems to be a bug that changes the name of the file, not an actual good reason.